src/Infrastructure/EventListener/ContextOutputCommandListener.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\EventListener;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\ContextFactoryInterface;
+use SprykerSdk\SdkContracts\Entity\MessageInterface;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ContextOutputCommandListener
```

Comment:
In not sure that it should be there. I can see there are console specific things. May be move it into the presentation console layer?